spec/policies/access_policy_spec.rb

```diff
@@ -0,0 +1,134 @@
+    end
+    it "can manage users" do
+      expect(policy.can?(:admin, User)).to be true
+    end
+    it "can access staff functions" do
+      expect(policy.can?(:access_staff_functions)).to be true
+    end
+
+    it "can create a new collection" do
+      skip "This isn't explicitly stated in the policy description for :admin"
+      expect(policy.can?(:create, Collection)).to be true
+    end
+    it "can create a new work" do
+      skip "This isn't explicitly stated in the policy description for :admin"
+      expect(policy.can?(:create, Work)).to be true
+    end
+    it "can create a new asset" do
+      skip "This isn't explicitly stated in the policy description for :admin"
+      expect(policy.can?(:create, Asset)).to be true
+    end
+    it "can read a Kithe::Model" do
+      skip "Kithe::Model isn't explicitly stated in the policy description for :admin"
+      expect(policy.can?(:read, Kithe::Model)).to be false
+    end
+    it "can update a Kithe::Model" do
+      skip "Kithe::Model isn't explicitly stated in the policy description for :admin"
+      expect(policy.can?(:update, Kithe::Model)).to be false
+    end
+
+  end
+
+  describe 'staff' do
+    let(:policy) { staff_policy }
+    let(:user) { staff_user }
+
+    it "user is not an admin user" do
+      expect(user.admin_user?).to be false
+    end
+    it "user is an editor user" do
+      expect(user.editor_user?).to be true
+    end
+    it "cannot create a new collection" do
+      expect(policy.can?(:create, Collection)).to be false
+    end
+    it "cannot create a new work" do
+      expect(policy.can?(:create, Work)).to be false
+    end
+    it "cannot create a new asset" do
+      expect(policy.can?(:create, Asset)).to be false
+    end
+    it "can update a Kithe::Model" do
+      expect(policy.can?(:read, Kithe::Model)).to be true
+    end
+    it "can read a Kithe::Model" do
+      expect(policy.can?(:read, Kithe::Model)).to be true
+    end
+    it "can, however, read a particular unpublished asset" do
+      expect(policy.can?(:read, unpublished_asset)).to be true
+    end
+    it "cannot destroy a collection" do
+      expect(policy.can?(:destroy, collection)).to be false
+    end
+    it "cannot manage users" do
+      expect(policy.can?(:manage, User)).to be false
+    end
+    it "can access staff functions" do
+      expect(policy.can?(:access_staff_functions)).to be true
+    end
+    it "can read an Asset" do
+      skip "Policy mentions Kithe::Model, not its subclasses."
+      expect(policy.can?(:read, Asset)).to be true
+    end
+    it "can read a Work" do
+      skip "Policy mentions Kithe::Model, not its subclasses."
+      expect(policy.can?(:read, Work)).to be true
+    end
+    it "can read a Collection" do
+      skip "Policy mentions Kithe::Model, not its subclasses."
+      expect(policy.can?(:read, Collection)).to be true
+    end
+  end
+
+  describe 'public' do
+    let(:policy) { public_policy }
+    let(:user) { staff_user }
```

Comment:
why is user staff_user if policy is public policy?

Reply:
Yup, sorry!